core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -1531,12 +1535,12 @@ private void prepareRowDescription() {
         sink.putNetworkShort((short) n);
         for (int i = 0; i < n; i++) {
             final int typeFlag = activeSelectColumnTypes.getQuick(i);
-            final int columnType = toColumnType(typeFlag == ColumnType.NULL ? ColumnType.STRING : typeFlag);
+            final int columnType = toColumnType(ColumnType.tagOf(typeFlag) == ColumnType.NULL ? ColumnType.STRING : typeFlag);
```

Comment:
this is small loop but we call tagOf excessively here, twice for the same value, perhaps we can optimise this a little bit